tests/integration/test.viewadapter.js

```diff
@@ -0,0 +1,128 @@
+/* To test these adapters, you can run
+ADAPTERS=memory CLIENT=selenium:firefox npm run test */
```

Comment:
`npm run test` and `npm test` are equivalent.

Reply:
i just added it from here in the document:
https://github.com/neighbourhoodie/pouchdb/blob/master/TESTING.md#L217